ramls/userSummary.raml

```diff
@@ -0,0 +1,34 @@
+#%RAML 1.0
+
+title: User Summary
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: User Summary
+    content: The JSON schemas will be generated into java objects
+
+types:
+  userSummary: !include userSummary.json
+  userSummaries: !include userSummaries.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+  language: !include raml-util/traits/language.raml
+  pageable:  !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/user-summary:
```

Comment:
We don't need API CRUD for UserSummary. Are you sure this section won't cause  CRUD endpoints to be created by RMB?